server/status.go

```diff
@@ -241,13 +242,19 @@ func parseInt64WithDefault(s string, defaultValue int64) (int64, error) {
 	return result, nil
 }
 
-// handleLocalLog returns the log entries parsed from the log file that occurred
-// after or on the query parameter "starttime" and before or on the query
-// parameter "endtime". Additionally, if a level is specified via the URL, the
-// log entries are scoped to that level. If no "starttime" is provided, then
-// a "starttime" of a day ago is used.  If no "endtime" is provided, the current
-// time is used. The cutoff for entries is defaults to defaultMaxLogEntries but
-// can be adjusted via the "max" query parameter.
+// handleLocalLog returns the log entries parsed from the log files stored on
+// the server. Log entries are returned in reverse chronological order with the
```

Comment:
fixed it to just `reverse chronological order`
not sure how descending works in this context